lib/capistrano/tasks/nvm.rake

```diff
@@ -28,7 +32,7 @@ namespace :nvm do
   task :wrapper do
     on release_roles(fetch(:nvm_roles)) do
       execute :mkdir, "-p", "#{fetch(:tmp_dir)}/#{fetch(:application)}/"
-      upload! StringIO.new("#!/bin/bash -e\nsource \"#{fetch(:nvm_path)}/nvm.sh\"\nnvm use $NODE_VERSION\nexec \"$@\""), "#{fetch(:tmp_dir)}/#{fetch(:application)}/nvm-exec.sh"
+      upload! StringIO.new("#!/bin/bash -e\nsource \"#{fetch(:nvm_path)}/nvm.sh\"\nnvm $NVM_VERB $NODE_VERSION\nexec \"$@\""), "#{fetch(:tmp_dir)}/#{fetch(:application)}/nvm-exec.sh"
```

Comment:
So here you can replace `$NVM_VERB` with `fetch(:nvm_verb)`

Reply:
Does the same go for `$NODE_VERSION` and `:nvm_node`?

Reply:
Oh crap, I guess I'm wrong here, this is of course the script that is created remotely. I'm sorry, it has been a long time that I worked on this project.

The only thing I'm now thinking is; wouldn't it be better to add a `nvm:install` task, and let that be idempotent, instead of having side effect when executing a node binary?

Reply:
>Oh crap, I guess I'm wrong here, this is of course the script that is created remotely.

Hmm... Thinking about it again, my current deploys rely on the destination servers being able to track changes to the `$NODE_VERISON` environment variable well after the `nvm-exec.sh` script is created and uploaded to them. So yeah that wouldn't work, at least not for me 😝 

>The only thing I'm now thinking is; wouldn't it be better to add a nvm:install task, and let that be idempotent, instead of having side effect when executing a node binary?

So like an `nvm:install` task that I could choose to invoke somewhere in my deploy, and then the wrapper script just runs `nvm use` before all of my commands?

While I think that this suggestion would probably be more elegant, `nvm install` is basically "Install if not installed already, then use." I think it makes sense to leverage the implicit behavior here since we basically get to let nvm worry about version management entirely just by configuring a ruby symbol.